control-plane/tests/test_phase19_operator_workflow_validation.py

```
@@ -0,0 +1,331 @@
+
+        created_alert = _load_wazuh_fixture("github-audit-alert.json")
+        restated_alert = deepcopy(created_alert)
+        restated_alert["id"] = "1731596200.1234568"
+        restated_alert["timestamp"] = "2026-04-05T12:35:00+00:00"
+        deduplicated_alert = deepcopy(restated_alert)
+
+        created = service.ingest_wazuh_alert(
+            raw_alert=created_alert,
+            authorization_header="Bearer reviewed-shared-secret",
+            forwarded_proto="https",
+            reverse_proxy_secret_header="reviewed-proxy-secret",
```

Ruff (0.15.9)

[error] Possible hardcoded password assigned to argument: "reverse_proxy_secret_header"

(S106)